rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,92 @@
+---
+feature: moderation team
+start-date: 2021-08-18
+author: tomberek
+co-authors: blaggacao
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/rfcs/pull/98
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive and evaluate
+applications to the team or alter the composition at any time with approval
```

Comment:
Maybe also extend invitations (≈ pre-approval of application before submission).

Reply:
something like this?
```suggestion
The Moderation Team is a volunteer group that may receive, invite, and evaluate
applications to the team or alter the composition at any time with approval
```

Reply:
Sounds good to me.